.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
```

Comment:
Is it necessary to have a turd file for this?

Reply:
It is the configuration file that is required to be present at the root of the repo. You just have to merge the PR with this file and then you can simply activate the analysis from the [dashboard](https://deepsource.io/gh/anacrolix/torrent/).

You can also use DeepSource to track test coverage, detect problems in Dockerfiles, etc. in addition to detecting issues in Go.

Reply:
Ok thanks.